mono/mini/driver.c

```diff
@@ -1767,7 +1764,7 @@ mono_jit_parse_options (int argc, char * argv[])
 			mono_enable_counters ();
 		} else if (strncmp (argv [i], "--stats=", 8) == 0) {
 			mono_enable_counters ();
-			set_stats_method_name (argv [i] + 8);
+			mono_stats_method_desc = parse_qualified_method_name(argv [i] + 8);
```

Comment:
What happens if `mono_stats_method_desc` is already set to a different value?

Reply:
If `mono_stats_method_desc` is already set to a value that points to valid memory on the heap, then there will be a leak. However, I don't want to arbitrarily free memory, so would the idiomatic solution here be to make `mono_stats_method_desc` static and only accessible through getters and setters?